### PR TITLE
Configurable email templates

### PR DESCRIPTION
**Pull request by rjkip**

### CODE REVIEW

File: src/Surfnet/Stepup/Configuration/Configuration.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ class Configuration extends EventSourcedAggregateRoot implements ConfigurationIn
     const CONFIGURATION_ID = '12345678-abcd-4321-abcd-123456789012';
 
     /**
-     * @var string
+     * @var array
      */
     private $configuration;
```
Comment (rjkip) [inline src/Surfnet/Stepup/Configuration/Configuration.php]: I believe you intended this to be a string. This still the case and what was your rationale?

**Comment by rjkip** [comment]
Any reason to ["update command handlers"](https://www.pivotaltracker.com/story/show/87500872)?

**Comment by rjkip** [comment]
What was the rationale for the separate `*UpdatedEvent`s? Was this for immediate consistency?

File: src/Surfnet/StepupMiddleware/ManagementBundle/Validator/EmailTemplatesConfigurationValidator.php [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+            Assert::keyExists(
+                $configuration[$templateName],
+                $this->requiredLocale,
+                "Required property '" . $this->requiredLocale . "' is missing",
+                $templatePropertyPath
+            );
+            Assert::string(
+                $configuration[$templateName][$this->requiredLocale],
+                'Property "en_GB" must have a string as value',
```
Comment (DRvanR) [inline src/Surfnet/StepupMiddleware/ManagementBundle/Validator/EmailTemplatesConfigurationValidator.php]: locale is configurable :wink:

**Comment by rjkip** [comment]
Better?

**Comment by DRvanR** [comment]
Seems that the entities and repositories are still in the CommandHandlingBundle rather than the ManagementBundle?

**Comment by rjkip** [comment]
I am-a sorry, I asleep. Come questo, signore?

**Comment by DRvanR** [comment]
:+1:

**Pull request merged**